frontend/shared/.eslintrc.js

```diff
@@ -20,5 +20,11 @@ module.exports = {
         'import/prefer-default-export': 'off',
       },
     },
+    {
+      files: ['*.components.ts'],
+      rules: {
+        'security/detect-non-literal-fs-filename': 'off',
```

Comment:
which file names are giving this errors?

Reply:
testcafe checks like:
```
        await t
          .expect(selectors.userName(name).exists)
          .ok(await getErrorMessage(t));
```
Because eslint thinks `exists` is related to use of `fs` but it isnt. Thus the rule is disabled for `*.components.ts` which are testcafe helper files.